apps/web/test/utils/bookingScenario/expects.ts

```diff
@@ -295,8 +295,6 @@ export function expectWebhookToHaveBeenCalledWith(
       }
       if (data.payload.responses !== undefined)
         expect(parsedBody.payload.responses).toEqual(expect.objectContaining(data.payload.responses));
-      const { responses: _1, metadata: _2, ...remainingPayload } = data.payload;
-      expect(parsedBody.payload).toEqual(expect.objectContaining(remainingPayload));
```

Comment:
Compare the 2 snippets at the end of the [Issue](https://github.com/calcom/cal.com/issues/15033) for why this is not needed.